Makefile

```diff
@@ -81,6 +81,11 @@ bin/dep-${DEP_VERSION}:
 .PHONY: vendor
 vendor: bin/dep ## Install dependencies
 	bin/dep ensure -v -vendor-only
+	@touch vendor/.auto
+
+vendor/.auto: Gopkg.lock bin/dep # install/update dependencies if needed
```

Comment:
This won't update anything, only install dependencies for the first time, same way as `make up`. Not sure I understand the added value.

Reply:
I mean updates the vendor folder to match the revisions/versions in the lock file. So you won't be confused about broken builds which would succeed if you would have run `make vendor -vendor-only` before the build.

Reply:
(Also see the other change in this commit)

Reply:
Okay, got it, thanks!